packages/components/resource-pages/CommonDetails.tsx

```diff
@@ -74,7 +74,7 @@ export const CommonDetails: React.FC<CommonDetailsSectionProps> = ({
         <div className="row">
           <div className="co-m-pane__body">
             <SectionHeading text={t('Conditions')} />
-            <Conditions conditions={(resource as any).status.conditions} />
+            <Conditions conditions={(resource as any)?.status?.conditions} />
```

Comment:
not sure why this PR is not showing any merge conflicts here: https://github.com/red-hat-storage/mcg-ms-console/pull/38 merged 3 hours ago...